src/src/attendances/attendances.controller.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { AttendancesService } from './attendances.service';
+import { CreateAttendanceDto } from './dto/create-attendance.dto';
+import { UpdateAttendanceDto } from './dto/update-attendance.dto';
+
+@Controller('attendances')
+export class AttendancesController {
+  constructor(private readonly attendancesService: AttendancesService) {}
+
+  @Post('/createattendances')
+  create(@Body() createAttendanceDto: CreateAttendanceDto) {
+    return this.attendancesService.create(createAttendanceDto);
+  }
+
+  @Get()
+  findAll(@Body() request: Record<string, any>) {
```

Comment:
you cann't pass request body in get request. if you are passing body make it Post request and add dto file validation here.